PWGHF/vertexingHF/AliAnalysisTaskCheckEvSel.cxx

```diff
@@ -295,6 +310,24 @@ void AliAnalysisTaskCheckEvSel::UserExec(Option_t */*option*/){
   Double_t zvSPD=vertexSPD->GetZ();
   Double_t zvTRK=vertex->GetZ();
 
+  if(fEnableVertexNtuple) {
+    Float_t wrej4ntuple = (Float_t)wrej;
+    if(isEvSel) wrej4ntuple = -1;
+    Float_t vertextype=0;
+    TString vertextitle = vertex->GetTitle();
```

Comment:
To be safe, it would be better to move the line
TString vertextitle = vertex->GetTitle();
inside the condition if(vertex).

Reply:
Hi Francesco, 
right thanks! I committed your suggestion!